test/expected/plan_expand_hypertable-10.out

```diff
@@ -2293,6 +2293,178 @@ INSERT INTO outer_join_2 VALUES(1,'a');
 
 DROP TABLE outer_join_1;
 DROP TABLE outer_join_2;
+-- test UNION
+SELECT time FROM regular_timestamptz UNION SELECT time FROM metrics_timestamptz ORDER BY 1;
+             time             
+------------------------------
+ Sat Jan 01 00:00:00 2000 PST
+ Sun Jan 02 00:00:00 2000 PST
+ Mon Jan 03 00:00:00 2000 PST
+ Tue Jan 04 00:00:00 2000 PST
+ Wed Jan 05 00:00:00 2000 PST
+ Thu Jan 06 00:00:00 2000 PST
+ Fri Jan 07 00:00:00 2000 PST
+ Sat Jan 08 00:00:00 2000 PST
+ Sun Jan 09 00:00:00 2000 PST
+ Mon Jan 10 00:00:00 2000 PST
+ Tue Jan 11 00:00:00 2000 PST
+ Wed Jan 12 00:00:00 2000 PST
+ Thu Jan 13 00:00:00 2000 PST
+ Fri Jan 14 00:00:00 2000 PST
+ Sat Jan 15 00:00:00 2000 PST
+ Sun Jan 16 00:00:00 2000 PST
+ Mon Jan 17 00:00:00 2000 PST
+ Tue Jan 18 00:00:00 2000 PST
+ Wed Jan 19 00:00:00 2000 PST
+ Thu Jan 20 00:00:00 2000 PST
+ Fri Jan 21 00:00:00 2000 PST
+ Sat Jan 22 00:00:00 2000 PST
+ Sun Jan 23 00:00:00 2000 PST
+ Mon Jan 24 00:00:00 2000 PST
+ Tue Jan 25 00:00:00 2000 PST
+ Wed Jan 26 00:00:00 2000 PST
+ Thu Jan 27 00:00:00 2000 PST
+ Fri Jan 28 00:00:00 2000 PST
+ Sat Jan 29 00:00:00 2000 PST
+ Sun Jan 30 00:00:00 2000 PST
+ Mon Jan 31 00:00:00 2000 PST
+ Tue Feb 01 00:00:00 2000 PST
+(32 rows)
+
+-- test UNION ALL
+SELECT time FROM regular_timestamptz UNION ALL SELECT time FROM metrics_timestamptz ORDER BY 1;
```

Comment:
It might be sufficient to just count the number of rows rather than show all rows. Many rows in the test output just make it harder to check for the error when something goes wrong.

Reply:
It is not sufficient, using count will mask errors in target list handling, it also changes the plan to a very different shape.